### PR TITLE
Remove quotes from the slug text

### PR DESCRIPTION
**Pull request by mmulich**
This removes quotes from the slug text. See openstax/cnx#389 for details.

### CODE REVIEW

**Review by karenc — APPROVED** [review]
Looks good to me :+1:

File: cnxcommon/urlslug.py [hunk 1]
```diff
@@ -32,7 +34,11 @@ def generate_slug(book_title, *other_titles):
         book_title = slugify(remove_html_tags(book_title))
         return book_title
```
Comment (BryanHouston) [inline cnxcommon/urlslug.py:35]: Should you also remove quotes from the book title?

**Review by BryanHouston — APPROVED** [review]
Just one comment/suggestion. Otherwise, looks good.

File: cnxcommon/urlslug.py (hunk 1 above)
Comment (mmulich) [reply cnxcommon/urlslug.py:35]: I'll leave that for later resolution as we aren't actually using the book slug ATM. It's not the intention of this change to DRY (Don't Repeat Yourself) things out. There is a way to write this function so it is DRY, which would correct the problem.

**Pull request merged**